lumper/src/vmm/src/lib.rs

```diff
@@ -160,6 +167,26 @@ impl VMM {
         Ok(())
     }
 
+    pub fn configure_console(
+        &mut self,
+        console_path: Option<String>
+    ) -> Result<()> {
+        if let Some(console_path) = console_path {
+            // We create the file if it does not exist, else we open
+            let file = OpenOptions::new().
+                append(true).
+                create(true).
+                open(console_path).
+                map_err(ConsoleError)?;
+
```

Comment:
You could simply do a

```rust
let console = File::create(&console_path).map_error(ConsoleError);
```

This will truncate the file if it already exists, which is the behaviour we're looking for.

Reply:
In fact, I thought the behavior we wanted was to make an update to the console file. I did the update with the new behavior.